testing/trino-product-tests/src/main/java/io/trino/tests/product/deltalake/TestDatabricksWithGlueMetastoreCleanUp.java

```diff
@@ -69,6 +71,12 @@ public void testCleanUpOldTablesUsingDelta()
 
     private void cleanSchema(String schema, long startTime, AWSGlueAsync glueClient)
     {
+        Database database = glueClient.getDatabase(new GetDatabaseRequest().withName(schema)).getDatabase();
+        if (database.getCreateTime().toInstant().isAfter(SCHEMA_CLEANUP_THRESHOLD)) {
```

Comment:
nit, pre-existing, SCHEMA_CLEANUP_THRESHOLD shouldn't be a static constant
it doesn't matter when class was loaded in current JVM. it matters when cleanup code was invoked.